lib/elixir_boilerplate/telemetry_ui/telemetry_ui.ex

```diff
@@ -0,0 +1,264 @@
34271 13.445838 17.430108 25.007417 11.549319 7.115151-3.619115 10.078654-11.387504 9.921651-19.81976-.162566-8.731042-17.034649-18.626155-20.022678-25.912745-1.992018-4.857728-2.923374-9.1913529-2.794067-13.0008765z" fill="url(#c)"/><path d="m41.199436 24.874043c5.220347 6.694959 6.358283 11.355459 3.413807 13.981497-4.416714 3.93906-15.217419 6.509155-21.936599 1.744215-4.479454-3.176628-6.174316-9.991206-5.084588-20.443737-1.849118 3.861723-3.412567 7.773671-4.690348 11.735849-1.27778 3.962178-1.650915 8.108529-1.119404 12.439052 1.601351 3.239683 5.494817 5.403396 11.680397 6.491139 9.278371 1.631615 18.060122.825407 23.95271-2.145065 3.928391-1.980314 5.786494-3.951651 5.574312-5.91401.141766-2.897853-.751847-5.656438-2.680832-8.275753-1.928988-2.619317-4.965472-5.823713-9.109455-9.613187z" fill="url(#d)"/><path d="m20.799251 18.189006c-.04364 4.835125 1.199603 9.431489 3.729718 13.789093 3.795174 6.536405 8.225212 12.995204 14.854367 18.348954 4.419436 3.569167 7.950747 4.722294 10.593929 3.459382-2.170994 3.88538-4.479397 5.78925-6.925211 5.711609-3.668718-.11646-8.142117-1.719788-15.309635-10.333032-4.778347-5.742163-8.047222-11.17387-9.806624-16.295121.279004-2.031676.574857-4.055285.887559-6.070826.312702-2.015542.971335-4.885561 1.975897-8.610059z" fill="url(#e)"/><path d="m32.011273 24.824412c.405511 3.938827 1.93822 10.239557 0 14.434591-1.938221 4.195036-10.890677 11.773476-8.419446 18.449432 2.47123 6.675957 8.493644 5.177168 12.271355 2.100547 3.777712-3.076623 5.799822-8.079349 6.248034-11.597523.448213-3.518171-1.072381-10.287708-1.56693-16.17596-.329698-3.9255-.106002-7.291185.671093-10.097054l-1.15758-1.456665-6.813456-2.017361c-1.092388 1.614111-1.503411 3.73411-1.23307 6.359993z" fill="url(#f)"/><path d="m34.443394 5.3148253c-2.205235.9318217-4.294586 2.7781986-6.268054 5.5391307-2.960203 4.141399-4.467906 6.623907-3.3519 14.833191.744003 5.472857 1.276531 10.50778 1.597582 15.104773l9.543032-27.726988c-.350835-1.412815-.642632-2.688766-.875391-3.8278548-.232758-1.1390887-.447848-2.446506-.645269-3.9222519z" fill="url(#g)"/><path d="m35.945755 13.009805c-2.422551 1.413992-4.299708 4.310913-5.631469 8.690763-1.331762 4.379852-2.550147 10.50277-3.655157 18.368756 1.47381-5.003069 2.451455-8.626771 2.932936-10.871105.722221-3.366501.968912-8.127131 2.886564-11.359156 1.278437-2.154684 2.434144-3.764436 3.467126-4.829258z" fill="#330a4c" fill-opacity=".316321"/><path d="m24.728788 59.937995c3.986659.569558 6.071303 1.075916 6.253931 1.519073.273942.664733-.504655 1.272785-2.717611.864177-1.475304-.272404-2.654077-1.06682-3.53632-2.38325z" fill="#fff"/><path d="m26.731652 5.3148253c-2.192807 2.6195801-4.092897 5.3967527-5.700271 8.3315187s-2.755892 5.124204-3.445555 6.568313c-.213753 1.077096-.318084 2.666371-.312993 4.767823.0051 2.101452.186856 4.438039.545295 7.009761.313817-5.035952 1.274508-9.924178 2.882072-14.664678 1.607565-4.740501 3.618049-8.7447464 6.031452-12.0127377z" fill="#ededed" fill-opacity=".603261"/></g></svg>
```

Comment:
Let’s use our own logo!

```suggestion
      <svg width="64" height="64" viewBox="0 0 266 266" xmlns="http://www.w3.org/2000/svg"><g fill-rule="nonzero" fill="none"><circle fill="#7F4BCB" cx="133" cy="133" r="133"/><path d="M135.08 60.197c-1.29-1.596-3.87-1.596-5.16 0C118.956 74.247 78 129.17 78 155.035 78 184.732 102.509 209 132.5 209s54.5-24.268 54.5-53.965c0-25.865-40.956-80.788-51.92-94.838Zm15.802 129.963c-.645.32-1.29.32-1.935.32-1.935 0-3.87-1.278-4.515-2.874-.968-2.555.322-5.11 2.58-6.387 19.026-7.663 17.414-30.654 17.092-30.974-.323-2.554 1.612-5.109 4.514-5.109 2.58-.32 5.16 1.597 5.16 4.47 1.29 10.219-2.58 32.252-22.896 40.554Z" fill="#3A186A"/></g></svg>
```